app/templates/dashboard.html

```diff
@@ -10,20 +10,28 @@ <h1 class="govuk-heading-xl">Your applications</h1>
     {% set ns = namespace(rows = []) %}
     {% for application in applications %}
         {% set status_class %}
-        {%  if application.status == 'COMPLETED' %}
-            govuk-tag--green
-        {%  elif application.status == 'NOT_STARTED' %}
-            govuk-tag--red
+            {%  if application.status == 'COMPLETED' %}
+                govuk-tag--green
+            {%  elif application.status == 'NOT_STARTED' %}
+                govuk-tag--red
+            {% endif %}
+        {% endset %}
+
+        {% if application.status == 'SUBMITTED' %}
+            {% set application_link = '<p class="govuk-body govuk-!-margin-bottom-0"></p>' %}
```

Comment:
Is there a way to just not include this rather than having an empty paragraph tag?